src/flows/pipes/QueryBuilder/CardList.tsx

```diff
@@ -50,6 +51,15 @@ const Card: FC<Props> = ({idx}) => {
   const [keySearches, setKeySearches] = useState([])
   const [valueSearches, setValueSearches] = useState([])
 
+  const allItems = useMemo(() => {
+    const selected = new Set(card.values.selected)
+    const remaining = card.values.results.filter(
```

Comment:
almost. the logic seems to be slightly reversed from this: filter out all results from _selected_ that exist in _results_, and prepend whatever is left from that to the _results_ list